moskito-backend/src/Entity/User.php

```diff
@@ -0,0 +1,164 @@
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=60)
+     */
+    private $firstName;
+
+    /**
+     * @ORM\Column(type="string", length=160)
+     */
+    private $lastName;
+
+    /**
+     * @ORM\Column(type="string", length=60)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $password;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=Campsite::class, inversedBy="users")
+     */
+    private $campsite;
+
+    /**
+     * @ORM\OneToMany(targetEntity=UserToken::class, mappedBy="user", orphanRemoval=true)
+     */
+    private $userTokens;
+
+    public function __construct()
+    {
+        $this->campsite = new ArrayCollection();
+        $this->userTokens = new ArrayCollection();
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getFirstName(): ?string
+    {
+        return $this->firstName;
+    }
+
+    public function setFirstName(string $firstName): self
+    {
+        $this->firstName = $firstName;
+
+        return $this;
+    }
+
+    public function getLastName(): ?string
+    {
+        return $this->lastName;
+    }
+
+    public function setLastName(string $lastName): self
+    {
+        $this->lastName = $lastName;
+
+        return $this;
+    }
+
+    public function getEmail(): ?string
+    {
+        return $this->email;
+    }
+
+    public function setEmail(string $email): self
+    {
+        $this->email = $email;
+
+        return $this;
+    }
+
+    public function getPassword(): ?string
+    {
+        return $this->password;
+    }
+
+    public function setPassword(string $password): self
+    {
+        $this->password = $password;
+
+        return $this;
+    }
+
+    /**
+     * @return Collection|Campsite[]
+     */
+    public function getCampsite(): Collection
+    {
+        return $this->campsite;
+    }
+
+    public function addCampsite(Campsite $campsite): self
+    {
+        if (!$this->campsite->contains($campsite)) {
+            $this->campsite[] = $campsite;
+        }
+
+        return $this;
+    }
+
+    public function removeCampsite(Campsite $campsite): self
+    {
+        $this->campsite->removeElement($campsite);
+
+        return $this;
+    }
+
+    /**
+     * @return Collection|UserToken[]
+     */
+    public function getUserTokens(): Collection
+    {
+        return $this->userTokens;
+    }
+
+    public function addUserToken(UserToken $userToken): self
+    {
+        if (!$this->userTokens->contains($userToken)) {
+            $this->userTokens[] = $userToken;
+            $userToken->setUser($this);
+        }
+
+        return $this;
+    }
+
+    public function removeUserToken(UserToken $userToken): self
+    {
+        if ($this->userTokens->removeElement($userToken)) {
+            // set the owning side to null (unless already changed)
```

Comment:
It looks all good to me, so I just leave a reminder here to delete Andreas' command at a later stage.

Reply:
oh yeah, that's auto-created comment from the relations wizard.